client/components/Menubar/MenubarMenu.jsx

```diff
@@ -0,0 +1,100 @@
+import classNames from 'classnames';
```

Comment:
Also found [this article that touches on implementing focus traps for dropdown menus](https://okenlabs.com/blog/accessibility-implementing-focus-traps/#:~:text=Understanding%20Focus%20Traps%20%23,components%20like%20modals%20or%20menus.). Was wondering if you feel that might be applicable here, or if it might be unnecessary?

Reply:
> I really, really love the changes here and the way this file is organized!
> 
> I know it's already within the tagged issue, but do you think it might be helpful to reference [the article you've used](https://blog.logrocket.com/building-accessible-menubar-component-react) as a comment here? Just came to mind since we have a few areas throughout the app that make these types of annotations!

absolutely -- will add it as a comment so that it's clearer as a reference.

> Also found [this article that touches on implementing focus traps for dropdown menus](https://okenlabs.com/blog/accessibility-implementing-focus-traps/#:~:text=Understanding%20Focus%20Traps%20%23,components%20like%20modals%20or%20menus.). Was wondering if you feel that might be applicable here, or if it might be unnecessary?

thanks for this! i feel that focus trapping is something i associate more with modals and dialogs, where users can't tab out of the element unless a certain action is performed. i think we also want to be able to tab to the next element from the menubar without completely restricting focus in the way that we would with modals / dialogs / overlays. i'll see if i can find a menubar implementation in the wild that uses focus trapping, but here it might not be unnecessary.

that being said, we will definitely need a level of focus management to handle tabbing into the component as well as focusing various submenus / menuitems -- in the examples i've found, this has been done with a [roving tabindex](https://www.w3.org/WAI/ARIA/apg/practices/keyboard-interface/#kbd_roving_tabindex).